app/.eslintrc.json

```diff
@@ -21,13 +21,7 @@
     "max-len": [2, 120, 4],
     "eol-last": 2,
     "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-unused-vars": [
```

Comment:
changed to warn as this makes a horrible dev experience as the page errors and doesn't load when editing and changing a file and you don't care about imports at that point

Reply:
we do get warnings when we compile and github will show warnings but wont fail pipelines.

Reply:
I see a single instance of that in the codebase - line 75 in packetColumns.tsx:
```ts
filterFn: (row, _columnId, filterValue) =>
```